app/assets/stylesheets/express_admin/shared/_alerts.sass

```diff
@@ -6,21 +6,52 @@
   transform: translateX(-50%)
   width: 400px
 
+  .success
+    background-color: #43ac6a
```

Comment:
We can add a `_variable.sass` for global declaration to use these colours across the express admin. You can include `_variable.sass` on the screen.sass. Make sure that it does not conflict with other global variables